packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -351,13 +384,99 @@ contract VotingV2 is
             uint256 totalSlashed =
                 ((noVoteSlashPerToken * (stakedAtRound - totalVotes)) / 1e18) +
                     ((wrongVoteSlashPerToken * (totalVotes - totalCorrectVotes)) / 1e18);
-            requestSlashingTrackers.push(
-                SlashingTracker(wrongVoteSlashPerToken, noVoteSlashPerToken, totalSlashed, totalCorrectVotes)
+
+            requestSlashingTrackers[i] = SlashingTracker(
+                wrongVoteSlashPerToken,
+                noVoteSlashPerToken,
+                totalSlashed,
+                totalCorrectVotes
             );
+
             lastRequestIndexConsidered = i + 1;
         }
     }
 
+    /****************************************
+     *       SPAM DELETION FUNCTIONS        *
+     ****************************************/
+
+    function signalRequestsAsSpamForDeletion(uint256[2][] memory spamRequestIndices) public {
```

Comment:
Could someone spam requests and then spam this with so many jumps that it breaks voters' ability to traverse the jumps? I think that's hard unless this function is less expensive than the jumping, but still might be possible since they could technically call this function multiple times.